cmd/arc_summary/arc_summary.py

```diff
@@ -228,6 +232,22 @@ def get_arc_summary(Kstat):
         'per': fPerc(target_size, target_max_size),
         'num': fBytes(target_size),
     }
+    output['arc_sizing']['meta_size'] = {
+        'per': fPerc(meta_size, meta_limit),
+        'num': fBytes(meta_size),
+    }
+    output['arc_sizing']['meta_limit'] = {
+        'per': fPerc(meta_limit, target_max_size),
+        'num': fBytes(meta_limit),
+    }
+    output['arc_sizing']['dnode_limit'] = {
+        'per': fPerc(dnode_limit, meta_limit),
+        'num': fBytes(dnode_limit),
+    }
+    output['arc_sizing']['dnode_size'] = {
+        'per': fPerc(dnode_size, dnode_limit),
+        'num': fBytes(dnode_size),
+    }
```

Comment:
nit: Let's reorder these lines and the ones in the `# ARC Sizing` section to be the same as the output order.   You'll want to make the same check to `arc_summary3.py`.

meta_limit
meta_size
dnode_limit
dnode_size